<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow
{
border-image: url(:/res/png/p1Back);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>1281</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>60</number>
     </property>
     <item>
      <widget class="ButtonCommon" name="btTest">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonCommon" name="btQc">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonCommon" name="btSet">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonCommon" name="btQuery">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonCommon</class>
   <extends>QPushButton</extends>
   <header location="global">buttoncommon.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
